gravitee-am-ui/src/app/domain/settings/audits/audits.component.html

```diff
@@ -128,7 +128,7 @@ <h1>Audit log</h1>
           <div style="padding-left: 35px; max-width: 95%">
             <div style="margin-bottom: 20px"><strong>Details</strong></div>
             <p *ngIf="row.accessPoint">
-              <span *ngIf="row.accessPoint.ipAddress"><strong>IP: </strong> {{ row.accessPoint.ipAddress }}</span>
+              <span *ngIf="row.accessPoint.ipAddress"><strong>IP: </strong> {{ row.accessPoint.ipAddress }} </span>
               <span *ngIf="row.accessPoint.userAgent"><strong>User agent: </strong> {{ row.accessPoint.userAgent }}</span>
```

Comment:
I suggest using padding instead of space. Someone can think it is a space put by a mistake.

for example <span style="padding-left: 5px" .... >